### PR TITLE
processed tests.basic.erc for Geth environment

### PR DESCRIPTION
**Pull request by IvanAdamovNeon**

### CODE REVIEW

File: contracts/EIPs/ERC173.sol [hunk 1]
```diff
@@ -47,7 +47,8 @@ contract ERC173Caller {
     }
 
     function transferOwnership(address newOwner) external {
-        address(erc173_address).call(abi.encodeWithSignature("transferOwnership(address)", newOwner));
+        (bool success, ) = address(erc173_address).call(abi.encodeWithSignature("transferOwnership(address)", newOwner));
+        require(success, "Ownership transfer failed"); // fails without this on Geth
```
Comment (kristinaNikolaevaa) [inline contracts/EIPs/ERC173.sol]: looks strange
does transfer itself work for geth and neon ?

File: integration/tests/basic/erc/test_EIP3541.py [hunk 2]
```diff
@@ -24,6 +24,7 @@ class TestRejectingContractsStartingWith0xEF:
     accounts: EthAccounts
 
     @pytest.mark.parametrize("data", BAD_CALLDATA)
+    @pytest.mark.bug  # Geth raises ValueError("{'code': -32000, 'message': 'invalid code: must not begin with 0xef'}")
```
Comment (kristinaNikolaevaa) [inline integration/tests/basic/erc/test_EIP3541.py:27]: please add this to created bug (or separate one)

File: contracts/EIPs/ERC173.sol (hunk 1 above)
Comment (IvanAdamovNeon) [reply contracts/EIPs/ERC173.sol]: works on both networks like this ☝️
but on Geth - fails without `require(success, "Ownership transfer failed");`

File: integration/tests/basic/erc/test_EIP3541.py (hunk 2 above)
Comment (IvanAdamovNeon) [reply integration/tests/basic/erc/test_EIP3541.py:27]: done
added here
https://neonlabs.atlassian.net/issues/NDEV-3073

File: contracts/EIPs/ERC173.sol (hunk 1 above)
Comment (IvanAdamovNeon) [reply contracts/EIPs/ERC173.sol]: @kristinaNikolaevaa 
changed so that it works on both envs seamlessly

**Review by kristinaNikolaevaa — APPROVED** [review]

**Pull request merged**